samples/OpenIdConnect.AzureAdSample/Startup.cs

```diff
@@ -70,7 +70,7 @@ public void ConfigureServices(IServiceCollection services)
                         var authContext = new AuthenticationContext(Authority, AuthPropertiesTokenCache.ForCodeRedemption(context.Properties));
 
                         var result = await authContext.AcquireTokenByAuthorizationCodeAsync(
-                            context.ProtocolMessage.Code, new Uri(currentUri), credential, Resource);
```

Comment:
I'd rather revert than add alias's, but I could be convinced either way, @PinpointTownes ?